File: app/src/main/java/com/example/numad25sp_hongguo/MainActivity.java
package com.example.numad25sp_hongguo;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import androidx.appcompat.app.AppCompatActivity;

public class MainActivity extends AppCompatActivity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        // Find the button after setting the content view
        Button aboutButton = findViewById(R.id.aboutButton);

        // Change the background color of About Me button
        aboutButton.setBackgroundColor(android.graphics.Color.parseColor("#ADD8E6"));

        // Set the click listener
        aboutButton.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent = new Intent(MainActivity.this, AboutMeActivity.class);
                startActivity(intent);
            }
        });

        // Calculator button setup
        Button calcButton = findViewById(R.id.calcButton);
        calcButton.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent = new Intent(MainActivity.this, CalculatorActivity.class);
                startActivity(intent);
            }
        });

        // Contact Button setup
        Button contactsButton = findViewById(R.id.contactsButton);
        contactsButton.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent = new Intent(MainActivity.this, ContactsActivity.class);
                startActivity(intent);
            }
        });

        // Prime Button setup
        Button primeButton = findViewById(R.id.primeButton);
        primeButton.setOnClickListener(new View.OnClickListener() {

            @Override
            public void onClick(View view) {
                Intent intent = new Intent(MainActivity.this, PrimeActivity.class);
                startActivity(intent);
            }
        });
    }
} 

File: app/src/main/java/com/example/numad25sp_hongguo/MainActivity.kt
package com.example.numad25sp_hongguo

import android.os.Bundle
import android.widget.Button
import android.widget.Toast
import androidx.appcompat.app.AppCompatActivity

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)  // Set the layout first

        // Find the button after setting the content view
        val aboutButton: Button = findViewById(R.id.aboutButton)

        // Change the background color of About Me button
        aboutButton.setBackgroundColor(android.graphics.Color.parseColor("#ADD8E6"))

        // Set the click listener
        aboutButton.setOnClickListener {
            Toast.makeText(
                this,
                "Name: Hong Guo\nEmail: guo.hong1@northeastern.edu",
                Toast.LENGTH_LONG
            ).show()
        }
    }
}